policy/modules/kernel/files.fc

```diff
@@ -259,6 +259,10 @@ ifndef(`distro_redhat',`
 /usr/src/kernels/.+/lib(/.*)?	gen_context(system_u:object_r:usr_t,s0)
 ')
 
+# Deploy Dell EMC PowerFlex - SDC(scini) driver configuration
+# https://cpsdocs.dellemc.com/bundle/VXF_DEPLOY/page/GUID-DC80A2E9-6B92-46F8-9276-1704B41E148E.html
+/usr/bin/emc/scaleio/[^/]*.ko  --		gen_context(system_u:object_r:modules_object_t,s0)
```

Comment:
This won't match ko files that are inside subdirectories... which could be a bit risky, since the documentation uses a regex that matches also those. I think we should just use the same regex as in the document (`.../scaleio/(.*)\.ko`). Also, the dot in `.ko` needs to be escaped ;)

Reply:
Thank you for quick feedback. I hope my latest commit should fix it. :)